alerts/ldap_password_spray.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, TermMatch
+
+
+class AlertLdapPasswordSpray(AlertTask):
+    def main(self):
+        # TODO: evaluate what the threshold should be to detect a password spray
+        search_query = SearchQuery(minutes=60)
```

Comment:
Suggestion: Make this configuration-based?  I did exactly this in my recent alert plugin:

https://github.com/mozilla/MozDef/pull/1294/files#diff-6368993e2d100a8f54f2e633d20463eeR179

See the config file at the top just takes a JSON object that gets "splatted" into keyword arguments.

Reply:
fixed, per your suggestion